scripts/percona-release.sh

```diff
@@ -313,14 +313,6 @@ function enable_alias {
   run_update
 }
 #
-function check_setup_command {
-  if [[ -n ${2} ]]; then
-    echo "* \"setup\" command does not accept additional options!"
-    show_help
-    exit 2
-  fi
-}
-#
 if [[ ${COMMANDS} != *${1}* ]]; then
```

Comment:
it is needed because we don't need to pass testing|release|experimental param/
User should correctly call setup command

Reply:
Fixed.